docs/book/src/developer/providers/migrations/v1.6-to-v1.7.md

```diff
@@ -28,3 +28,9 @@ maintainers of providers and consumers of our Go API.
 * Patch helper now return error with enough error context (https://github.com/kubernetes-sigs/cluster-api/pull/9946). It is recommended to remove redundant error context on call sites if applicable.
 
 ### Suggested changes for providers
+
+* [MachinePools are now enabled by default](https://github.com/kubernetes-sigs/cluster-api/pull/10141) and the feature flag is marked as `beta` instead of `alpha`. These values will need to be updated. If not, the following error will result:
```

Comment:
nit
```suggestion
* [MachinePools are now enabled by default](https://github.com/kubernetes-sigs/cluster-api/pull/10141) and the feature flag is marked as `beta` instead of `alpha`. If you are re-defining feature flags in your codebase, these values will need to be updated accordingly. If not, the following error will result:
```